lib/contentful.js

```diff
@@ -211,3 +214,60 @@ module.exports.renderMessageForPhoenixCampaign = function (phoenixCampaign, msgT
       .catch(err => reject(err));
   });
 };
+
+/**
+ * Returns all rendered messages for given phoenixCampaign, using defaults when no override defined.
+ */
+module.exports.renderAllMessagesForPhoenixCampaign = function (phoenixCampaign) {
+  logger.debug(`renderAllMessagesForPhoenixCampaign:${phoenixCampaign.id}`);
+
+  return new Promise((resolve, reject) => {
+    const campaignFieldNames = Object.keys(contentfulCampaignFields);
+    const campaignContent = {};
+    const result = {};
```

Comment:
@sergii-tkachenko Not sure what you're asking here -- this function's only called from within `fetchCampaign` in `routes/campaigns`...